src/plugins/intel_npu/src/utils/include/intel_npu/utils/remote_tensor_type/remote_tensor_type.hpp

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+namespace intel_npu {
+
+enum class RemoteMemoryType { EMPTY, L0_INTERNAL_BUF, SHARED_BUF };
+
+enum class RemoteTensorType { INPUT, OUTPUT, BINDED };
```

Comment:
I was thinking is easier to follow this way. I'm not having a good explanation here.

Reply:
Yeah, the include path doesn't look great, but otherwise we need to use something as ` ov::intel_npu::MemType::` all the time. Can not say which one is better or worse.